package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "aperture.js",
-  "version": "0.1.0",
+  "version": "0.2.0",
```

Comment:
I know, I missed that by mistake. I bumped the version locally to be able to test it with kap